scripts/modules_auto_pr/modules_auto_pr.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+####################################################################################################
+#                                            VARIABLES                                             #
+####################################################################################################
+
+# Variables (replace these with your actual values)
+DATE=$(date +"%Y-%m-%d_%Hh%M")
+REPO_URL="git@github.com:lbarraga/token-test.git"
+BASE_BRANCH="main"
+BRANCH_NAME="new-feature-branch_$DATE"
+REPO_NAME="token-test_$DATE"
+REPO_PATH="/tmp/$REPO_NAME"
+FILE_NAME="newfile.txt"
+COMMIT_MESSAGE="Add newfile.txt"
+PR_TITLE="Add new file as part of feature"
+PR_BODY="This PR adds a new file to the repository."
+
+# Use GitHub CLI for authentication
+GH_TOKEN="SAMPLE_TOKEN" # Test token, will not be used in production
+
+echo_info()    { echo -e "\e[32m$0: [INFO] $1\e[0m"; }
+echo_warning() { echo -e "\e[33m$0: [WARNING] $1\e[0m"; }
+echo_error()   { echo -e "\e[31m$0: [ERROR] $1\e[0m"; }
+echo_error_exit() {
+  echo_error "$1"
+  rm -rf "$REPO_PATH"
+  exit 1
+}
+
+####################################################################################################
+#                                             SCRIPT                                               #
+####################################################################################################
+
+echo_info "Logging in to GitHub..."
+gh auth login --with-token <<< "$GH_TOKEN" || echo_error_exit "Failed to log in to GitHub"
```

Comment:
you can have bash fail the whole script once something fails; and run with `set -x` to get an idea how far you got.

so please get rid of the `|| ...` stuff

Reply:
done in [cb25ef9](https://github.com/hpcugent/vsc_user_docs/pull/680/commits/cb25ef95fb784dd4e68832b5b1d35a65901ae70b)